chromium_src/extensions/browser/extension_util.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Disable extensions for tor OTR profile
+#define BRAVE_IS_INCOGNITO_ENABLED \
+  if (context->IsTor()) {          \
```

Comment:
Please check if component extensions like shields is still working.

Reply:
fixed in 34551d557a3a99b16fb21ec7c4dd581cb225b01e